models/system/setting_test.go

```diff
@@ -37,6 +37,11 @@ func TestSettings(t *testing.T) {
 	err = system.SetSetting(updatedSetting)
 	assert.NoError(t, err)
 
+	// make sure it's not cached
+	updatedSetting = &system.Setting{SettingKey: keyName, SettingValue: "100", Version: newSetting.Version}
+	err = system.SetSetting(updatedSetting)
+	assert.NoError(t, err)
```

Comment:
Isn't there an `assert` missing?
At which place do you test that it is not cached?

Reply:
This lines removed. Please review again.

Reply:
This lines removed. Please review again.